doc/development.md

```diff
@@ -35,24 +35,46 @@ Create virtual environment for vdsm:
 
 ## Building Vdsm
 
-To configure sources (run `./configure --help` to see configuration options):
+To configure sources (create Makefiles and configure script): 
 
     git clean -xfd
     ./autogen.sh --system --enable-timestamp
-    make
 
-To test Vdsm (refer to tests/README for further tests information):
+(It is recommended to start fresh after an update, from the previous step, before
+building RPMs, because the generated Makefile also contains the release number.)
```

Comment:
It is also not clear what is "start fresh" and "the previous step", and it will break easily when someone will add a new step before this step.

It think it is enough to explain the needed steps, like:

    Configure vdsm:
  
        ./autogen.sh ...

And add a note right after why this is needed:

   NOTE: if you don't configure before building rpms you may
   get packages with wrong release string, and upgrading will
   fail.